Essentials/src/main/java/com/earth2me/essentials/commands/Commandessentials.java

```diff
@@ -82,7 +82,6 @@ public class Commandessentials extends EssentialsCommand {
         "DiscordSRV", // potential for issues if EssentialsXDiscord is installed
 
         // Brain-dead chat signing bypass that break EssentialsChat
-        "AntiPopup",
```

Comment:
`versionPlugins` just shows the version of a plugin in `/ess version` and doesn't warn the user at all (for example we have DiscordSRV here so its easy for us to identify when discord is getting ratelimited), and I'd like to keep AntiPopup here just in case.

Reply:
Could we move it a line above 😂, I don't see what's brain-dead in my plugin.

Reply:
Comment can be changed to `Chat signing bypass plugins`

Reply:
Please recheck my comment, would you still want my plugin to be there?

Reply:
Please read https://github.com/EssentialsX/Essentials/pull/5037#discussion_r939704913

Reply:
Yeah I wasn't sure exactly what that list did when I was removing it. I've re-added it to the list and changed the comment to `Chat signing bypass plugins that can potentially break EssentialsChat`, as this seems to make the most sense considering what it does.